src/base-element.js

```diff
@@ -172,8 +172,7 @@ export class BaseElement {
    * @return {!./layout-rect.LayoutRectDef}
    */
   getLayoutBox() {
-    return this.element.getResources().getResourceForElement(
-        this.element).getLayoutBox();
+    return this.element.getLayoutBox();
```

Comment:
is this a DOM api? i think this should be from resources-impl - like how it was before?